tests/tiup/upgrade-from-v2.sh

```diff
@@ -79,7 +79,8 @@ function migrate_in_v2 {
 	echo "check locks"
 	run_dmctl_with_retry $CUR_VER "show-ddl-locks" "no DDL lock exists" 1
 
-	tiup dmctl:$CUR_VER --master-addr=master1:8261 stop-task $TASK_NAME
+	export DM_MASTER_ADDR="master1:8261"
+	tiup --binpath="./bin/dmctl" dmctl:$CUR_VER stop-task $TASK_NAME
```

Comment:
Can we pass the test if remove binpath? We patch tiup with locally build version at L54.

Reply:
Seems fail in above test. 🤔

Reply:
seems tiup still use  nightly version dmctl after update, how can we use the new one？
`Starting component `dmctl`: /root/.tiup/components/dmctl/v5.0.0-nightly-20210524/dmctl/dmctl stop-task`

Reply:
I will have a look later.

Reply:
Change this [CUR_VER](https://github.com/pingcap/dm/blob/0f3a7755d0e69693a99821c2981e9bbba0747bff/tests/tiup/upgrade-from-v2.sh#L49) to PRE_VER. 🤣